### PR TITLE
feat(toolbar): add linkout to gh code search from feature flag panel

### PR DESCRIPTION
**Pull request by michellewzhang**
clicking on a feature flag now links to code search for `sentry-options-automator` and `sentry`:

https://github.com/user-attachments/assets/914b7166-dec9-4bfd-b7a7-7512b46abbba

### CODE REVIEW

File: static/app/utils/useDevToolbar.tsx [hunk 1]
```diff
@@ -25,6 +25,8 @@ export default function useDevToolbar({enabled}: {enabled: boolean}) {
         projectId: 11276,
         projectSlug: 'javascript',
         featureFlags: organization.features,
+        featureFlagTemplateUrl: flag =>
+          `https://github.com/search?q=repo%3Agetsentry%2Fsentry-options-automator+OR+repo%3Agetsentry%2Fsentry+${flag}&type=code`,
```
Comment (ryan953) [inline static/app/utils/useDevToolbar.tsx:29]: Passing a function is a good way to inject the logic

File: static/app/components/devtoolbar/hooks/useConfiguration.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ const context = createContext<Configuration>({
   projectId: 0,
   projectSlug: '',
   featureFlags: [],
+  featureFlagTemplateUrl: _flag => '',
```
Comment (ryan953) [inline static/app/components/devtoolbar/hooks/useConfiguration.tsx:13]:
```suggestion
  featureFlagTemplateUrl: undefined,
```

File: static/app/components/devtoolbar/types.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ export type Configuration = {
   projectSlug: string;
   SentrySDK?: typeof SentrySDK;
   domId?: string;
+  featureFlagTemplateUrl?: (flag: string) => string;
```
Comment (ryan953) [inline static/app/components/devtoolbar/types.ts:14]:
```suggestion
  featureFlagTemplateUrl?: undefined | (flag: string) => string;
```

File: static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx [hunk 4]
```diff
@@ -42,7 +43,22 @@ export default function FeatureFlagsPanel() {
           .map(flag => {
             return (
               <Cell key={flag} style={{alignItems: 'flex-start'}}>
-                {flag}
+                {featureFlagTemplateUrl ? (
+                  <ExternalLink
+                    style={{textDecoration: 'inherit', color: 'inherit'}}
+                    href={featureFlagTemplateUrl(flag)}
```
Comment (ryan953) [inline static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx:49]: Might need to be {featureFlagTemplateUrl(flag) ? (`

so we catch a case where the empty string is returned. or maybe the return type should be `string | undefined` for the function? idk.

Comment (ryan953) [reply static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx:49]: not super important tho

**Review by ryan953 — APPROVED** [review]

File: static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx (hunk 4 above)
Comment (michellewzhang) [reply static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx]: just modified !

**Review by ryan953 — APPROVED** [review]

**Pull request merged**